scripts/generate_content.sh

```diff
@@ -5,7 +5,7 @@ cd ./web
 ../vendor/bin/drush --nocolor -y en devel_generate
 
 echo "Creating content type subjects"
-../vendor/bin/drush --nocolor generate-terms subjects 20 --kill
+../vendor/bin/drush migrate-import jcms_subjects_json
```

Comment:
This change would have caught this bug. We introduce new MSA using the migrate feature so if we make the same error again then this will trigger an error in `Drupal\Tests\jcms_rest\Functional\RecursiveEndpointValidatorTest::testValidEndpointsRecursively`:

```
$ ./project_tests.sh
coder_sniffer
Config value "installed_paths" updated successfully; old value was "vendor/drupal/coder/coder_sniffer"
PHPUnit tests
PHPUnit 4.8.36 by Sebastian Bergmann and contributors.

Runtime:	PHP 7.1.8-2+ubuntu16.04.1+deb.sury.org+4 with Xdebug 2.5.0
Configuration:	/var/www/journal-cms/phpunit.xml.dist

............................E.................................... 65 / 78 ( 83%)
.............

Time: 2.22 minutes, Memory: 14.00MB

There was 1 error:

1) Drupal\Tests\jcms_rest\Functional\RecursiveEndpointValidatorTest::testValidEndpointsRecursively with data set #0 ('/subjects', 'id', 'application/vnd.elife.subject...t+json', 'application/vnd.elife.subject+json')
InvalidArgumentException: json_decode error: Syntax error

/var/www/journal-cms/vendor/guzzlehttp/guzzle/src/functions.php:304
/var/www/journal-cms/src/modules/jcms_rest/tests/src/Functional/FixtureBasedTestCase.php:86
/var/www/journal-cms/src/modules/jcms_rest/tests/src/Functional/RecursiveEndpointValidatorTest.php:154

FAILURES!
Tests: 78, Assertions: 380, Errors: 1.
```

Reply:
cc @giorgiosironi

Reply:
As far as I understand this uses the real subjects (same as `prod`) rather than generated ones, so it' more high-fidelity than before.